packages/slo-stat-bot/test/fixtures/events/issue_slo_rules/invalid_slos/incorr_time_format.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "appliesTo": {
+            "issues": true
+        },
+        "complianceSettings": {
+            "responseTime": 0,
+            "resolutionTime": "1h 40m"
+        }
+    }
+ ]
+ 
+ 
```

Comment:
nit: we should make our linter catch multiple training newlines in files like this.